lgc/test/lgcdis-fail.txt

```diff
@@ -0,0 +1,5 @@
+; RUN: ( lgcdis %s || true ) 2>&1 | FileCheck %s
```

Comment:
Use `not` if the expected exit code is non-zero.

Reply:
That didn't work for me because the exit code is actually 128 or something, and "not" does not change that to 0.